api/controllers/blobs/create.js

```diff
@@ -32,17 +30,11 @@ module.exports = {
   fn: async ({ source, plugin, configs }) => {
     sails.log("Creating blob");
 
-    const blobBase64 = crypto
-      .createHash("sha256")
-      .update(Snippet.salt)
-      .update(JSON.stringify({ source, plugin, configs }))
-      .digest("hex");
-
     // Check for Blob and add if not present
-    const newBlob = await Blobs.findOrCreate(
-      { base64BlobKey: blobBase64 },
-      { base64BlobKey: blobBase64 }
-    );
+    const newBlob = await Blobs.create().fetch();
+    
+    // Add shareble link to response body
+    newBlob.url = `http://localhost:1337/share/${newBlob.id}`;
```

Comment:
We might want to refactor the `http.localhost` to somewhere else like an environment variable just so that it stays consist across the project

Reply:
We can use something like [sails.config.http](https://sailsjs.com/documentation/reference/configuration/sails-config-http)

Reply:
I'll look into it

Reply:
Changed it to `sails.config.port`. Production will look different for sure. Still have to figure that out